internal/transformers/typeeraser.go

```diff
@@ -100,6 +100,10 @@ func (tx *TypeEraserTransformer) visit(node *ast.Node) *ast.Node {
 			// TypeScript module declarations are elided if they are not instantiated or have no body
 			return nil
 		}
+		if ast.IsModuleDeclaration(tx.parentNode) && tx.parentNode.AsModuleDeclaration().Body == node {
```

Comment:
Is this necessary? I would think this would be handled in runtimesyntax.go.

Reply:
I wasn't sure whether nested namespaces were "type erased" syntax or runtime syntax. Putting it in the type eraser would, I guessed, make sure that it was back to the Strada-expected form both for .js emit and .d.ts emit.

Like I said in the runtimesyntax comment, maybe it's better to expect the .d.ts emit to change (or the .d.ts emitter to change). What do you think?

Reply:
for `export namespace a.b ...`? I think I would expect our .d.ts to continue emit `export namespace a.b ...`.

Reply:
hmm. OK. I can remove the changes in both typeeraser and runtimesyntax then.